packages/dds/merge-tree/src/zamboni.ts

```diff
@@ -0,0 +1,183 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
+import { MergeTree } from "./mergeTree";
+import { MergeTreeMaintenanceType } from "./mergeTreeDeltaCallback";
+import { IMergeBlock, IMergeNode, ISegment, MaxNodesInBlock } from "./mergeTreeNodes";
+import { matchProperties } from "./properties";
+
+export class Zamboni {
```

Comment:
i don't think a class is necessary here. there the constructor is empty, and there is no members. i think these should just be free functions, e.g. `export function zamboniSegments(mergeTree: MergeTree, zamboniSegmentsMaxCount = MergeTree.zamboniSegmentsMaxCount)`

private function can then just not be exported `function underflow(node: IMergeBlock)`

Reply:
i think doing this will also get rid of the slight bundle since increase this change currently causes